sdm/src/test/java/com/sap/cds/sdm/handler/TokenHandlerTest.java

```diff
@@ -288,44 +288,42 @@ public void testGetAccessToken() {
   }
 
   @Test
-  public void testGetAccessTokenNoCache() throws IOException {
-    // Mock SDMCredentials
+  public void testGetAccessTokenNoCache() {
     SDMCredentials mockSdmCredentials = Mockito.mock(SDMCredentials.class);
     when(mockSdmCredentials.getClientId()).thenReturn("mockClientId");
     when(mockSdmCredentials.getClientSecret()).thenReturn("mockClientSecret");
     when(mockSdmCredentials.getBaseTokenUrl()).thenReturn("https://mock.url");
 
-    Cache<String, String> mockCache = Mockito.mock(Cache.class);
-    when(mockCache.get(any())).thenReturn(null); // Cache is empty
-
+    String token =
+        "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6ImpvaG4uZG9lQGV4YW1wbGUuY29tIiwic3ViIjoiMTIzNDU2Nzg5MCIsIm5hbWUiOiJKb2huIERvZSIsImlhdCI6MTY4MzQxODI4MCwiZXhwIjoxNjg1OTQ0MjgwLCJleHRfYXR0ciI6eyJ6ZG4iOiJ0ZW5hbnQifX0.efgtgCjF7bxG2kEgYbkTObovuZN5YQP5t7yr9aPKntk";
```

Comment:
Shall we refrain from hardcoding the token ? This is for test and will be expired key anyway. So why not just have placeholder string instead ?

Reply:
we decode the token and fetch some details and need the hard coded token for the same